modules/local/ucsc_bed12tobigbed.nf

```diff
@@ -0,0 +1,42 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+def VERSION = '377'
+
+process UCSC_BED12TOBIGBED {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:'bigBed', publish_id:'meta.id') }
+
+    conda     (params.enable_conda ? "bioconda::ucsc-bedtobigbed=377" : null)
+    container "quay.io/biocontainers/ucsc-bedtobigbed:377--h446ed27_1"
+    if ({ task.exitStatus in [255] }) { errorStrategy = 'ignore' }
```

Comment:
Would this not be better in `base.config`?

Reply:
The reason this is here is that base.config has already defined  `errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }`

Reply:
OK, but you can add additional custom exit codes in base.config on a per process basis:

DSL1 example: https://github.com/nf-core/eager/blob/cc66639a8a2710f0de6aef5ae28e1614a5a4e3e2/conf/base.config#L93-L95

I suggest to do this to keep it all in one place, rather than spread all over the place.